bin/concurrently.ts

```diff
@@ -151,7 +151,7 @@ const args = yargs(argsBeforeSep)
         },
         'timestamp-format': {
             alias: 't',
-            describe: 'Specify the timestamp in moment/date-fns format.',
+            describe: 'Specify the timestamp in Unicode LDML format.',
```

Comment:
While this is technically correct, I wonder how many people won't know how to find usage examples for LDML.

For instance, for me, Google returns _one result_ for "unicode ldml" in the first page that is more "straight to the point" IMO: 
https://stackoverflow.com/q/39904392/2083599

Reply:
Good point! Should we explicitly link https://unicode.org/reports/tr35/tr35-dates.html#Date_Format_Patterns in this description?

Reply:
Probably, but i'm worried it might not be formatted well by yargs.

Something like #445 (or better) is looking promising now...